ui/utils.go

```diff
@@ -0,0 +1,73 @@
+package ui
+
+import (
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/RocketChat/Rocket.Chat.Go.SDK/models"
+	"github.com/joho/godotenv"
+)
+
+func goDotEnvVariable(key string) string {
+
+	// load .env file
+	err := godotenv.Load(".env")
+
+	if err != nil {
+		log.Fatalf("Error loading .env file")
+	}
+
+	return os.Getenv(key)
+}
```

Comment:
Load the file once in `init` - then maybe get each key just once and export. Like,
```go
func init() {
    if err := godotenv.Load(".env"); err != nil {
        // handle error
    }
}
var SITE_URL = os.Getenv("DEV_SERVER_URL")
var EMAIL = os.Getenv("EMAIL")
var PASS = os.Getenv("PASS")
```
now just reuse those.

Reply:
yes it shouldn't be loaded everytime I call the function. I'll update it.